test/system/test_cvp_client.py

```diff
@@ -31,6 +31,13 @@
 #
 
 ''' System test for the CvpClient class
+
+    Requirements for CVP Node:
+    1) Just need one node for test with the following account:
```

Comment:
The error is obvious is the CVP username or password are not correct. If the device does not have username and password the same as the CVP user used then the error is not obvious as it is found in the task log. Documented this case.